openedx/features/caliper_tracking/transformers/enrollment_transformers.py

```diff
@@ -34,17 +34,35 @@ def edx_course_enrollment_deactivated(current_event, caliper_event):
     :param caliper_event: log containing both basic and default attribute
     :return: final created log
     """
+    course_id = current_event['context']['course_id']
+
     caliper_object = {
-        'course_id': current_event['context']['course_id'],
-        'user_id': caliper_event['user_id'],
-        'mode': current_event['event']['mode']
+        'id': 'http://localhost:18000/courses/{}/about'.format(course_id),
+        'type': 'Membership'
     }
+
     caliper_event.update({
         'object': caliper_object,
-        'type': 'Enrollment',
-        'action': 'Deactivated'
+        'type': 'Event',
+        'action': 'Deactivated',
     })
-    caliper_event['actor']['type'] = 'Person'
+
+    caliper_event['actor'].update({
+        'type': 'Person',
+        'name': current_event['username']
+    })
+
+    caliper_event['referrer'].update({
```

Comment:
here it is only a one key to update 
so I think it will be better to do this as
caliper_event['referrer']['type'] = 'WebPage'